testdata/Off/boxcube.off

```diff
@@ -0,0 +1,108 @@
+Source:
+This OFF model was downloaded from http://www.holmes3d.net/graphics/offfiles/
+and was not created by the developers of the Open Infra Platform.
+
```

Comment:
I believe that this makes it an invalid OFF file. Shouldn't this be marked as "comments" in OFF format?

Reply:
Resolved: c45de96